### PR TITLE
Rewrite outdated supported dtypes

### PR DESCRIPTION
**Pull request by NripeshN**
# PR Description 

## Related Issue 

Close #

## Checklist 

- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

**Comment by NripeshN** [comment]
ivy-gardener

File: ivy/functional/frontends/jax/numpy/linalg.py [hunk 1]
```diff
@@ -81,6 +82,17 @@ def pinv(a, rcond=None):
...
+        "2.5.1 and below": {
+            "cpu": (
+                "float32",
+                "float64",
+            )
+        }
+    },
+    "jax",
+)
 def norm(x, ord=None, axis=None, keepdims=False):
```
Comment (AnnaTz) [inline ivy/functional/frontends/jax/numpy/linalg.py:96]: The jax version is different, and also I don't think we need the device.
```
@with_supported_dtypes(
    {"0.4.14 and below": ("float32","float64")},
    "jax",
)
```

**Review by AnnaTz — CHANGES_REQUESTED** [review]

**Comment by NripeshN** [comment]
ivy-gardener

**Comment by abdllahdev** [comment]
Hi, there is a missing import in this file [ivy/functional/frontends/jax/numpy/linalg.py](https://github.com/unifyai/ivy/pull/22565/commits/422bd0ba76f2b3bd17b2468ac144dc3fe82958da#diff-4e9d5ec55b3112ae12f59f6a33c6157d51020ba720f35934cfd9cfbb86624cab). This PR #22565 should solve the issue.

**Comment by abdllahdev** [comment]
@AnnaTz @NripeshN 
Hi again 😅, while making the previous PR, I found that the eigvals function in this file is not using the write function. Instead it's using `eigh` also the unit test for this function is not implemented right so I updated that too. You can find these updates here #22568. Would love it if you can review it sometime.

**Pull request merged**